docs/Creating-Components.md

```diff
@@ -1,150 +0,0 @@
-
-NB: we try to build "AMP-first", but this is difficult in practice because we are unable to inject the AMP dependency into Storybook. For now, check the AMP output manually in Part 2 of the process. Simorgh currently only supports one 'view' of components; components which require a different implementation for AMP must wait for the outcome of our [AMP/canonical component investigation](https://github.com/bbc/simorgh/issues/884).
-
-Once you're happy with how your component looks (in all its variants), work on your index.test.jsx file to take snapshots of them. If the structure of the component changes in the future, the unit test suite will fail.
-
-Once done, make a PR - this first step will need code review and UX QA.
-
-Example implementations below:
-
-### index.jsx
-```js
-import styled from 'styled-components';
-
-const Blockquote = styled.blockquote`
-  background: #f9f9f9;
-`;
-
-export default Blockquote;
-```
-
-### index.stories.jsx
-```js
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-import Blockquote from './index';
-
-storiesOf('Blockquote', module).add('default - quote only', () => <Blockquote>To be or not to be, that is the question</Blockquote>);
-storiesOf('Blockquote', module).add('quote with cite', () => (
-  <Blockquote cite="William Shakespeare">To be or not to be, that is the question</Blockquote>
-));
-```
-
-### index.test.jsx
-```js
-import React from 'react';
-import { shouldMatchSnapshot } from '../../testHelpers';
-import Blockquote from './index';
-
-describe('Blockquote', () => {
-  shouldMatchSnapshot(
-    'should render correctly',
-    <Blockquote>To be or not to be, that is the question</Blockquote>,
-  );
-});
-```
-
-## Part 2: Create container component and fixture data
-Once your component front-end is merged, you'll need to integrate it into the renderer.
-
-### Update the schema
-data/schema.yaml describes the Article API definition for web. We need to make it aware of our new component.
-
-Your component is more than likely a new 'block' in the data feed, so you'll need to add it to the array of which blocks the application should validate:
-
-```yaml
-    blocks:
-      type: object
-      items:
-        oneOf:
-          - $ref: '#/components/schemas/altText'
-          ... etc ...
-          - $ref: '#/components/schemas/[your component name]'
-      minItems: 2
-```
-
-You'll also need to define the block subtype itself:
-
-```yaml
-    blockquote:
-      type: object
-      required:
-        - model
-        - type
-      properties:
-        model:
-          properties:
-            blocks:
-              $ref: '#/components/schemas/blocks'
-          type: object
-        type:
-          enum:
-            - blockquote
-          type: string
-```
-
-The schema check currently only happens on local data. Eventually, we hope to run the schema validator at runtime.
-
-### Update some local fixture data
-Pick a JSON file under `data/news/articles/[id].json`, and:
```

Comment:
There's scope ambiguity in this suggestion, which sections should be kept? I suggset the location is no best under Fixture data in contributing.

Reply:
Contributing.md - under a new heading `## Local fixture data`, which can have the sections:
`### Updating local fixture data`
`### Updating the schema`

Reply:
Actually, I see you've already got a `## Fixture data` section, so it can live there https://github.com/bbc/simorgh/pull/2870/files#diff-6a3371457528722a734f3c51d9238c13R105

Reply:
Done